docs/Configuration.md

```diff
@@ -358,13 +358,19 @@ Default: `undefined`
 This option allows the use of a custom global setup module which exports an
 async function that is triggered once before all test suites.
 
+Jest's configuration object is passed as a parameter to the `globalSetup`
+function.
+
 ### `globalTeardown` [string]
 
 Default: `undefined`
 
 This option allows the use of a custom global teardown module which exports an
 async function that is triggered once after all test suites.
 
+Jest's configuration object is passed as a parameter to the `globalTeardown`
```

Comment:
It's not, strictly speaking, Jest configuration. It's a part of config, that's responsible for globally available options, for every `project` out there. I think we should make it clear and link e.g. to type definition.